File: Modul 02/01/PRAK201-2410817310020-MRizkiDinar.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    char arr[7][2][100] = {
        {"Nama                      :", ""},
        {"NIM                       :", ""},
        {"Kelas Paralel             :", ""},
        {"Tempat/Tanggal Lahir      :", ""},
        {"Alamat                    :", ""},
        {"Hobby                     :", ""},
        {"No. HP                    :", ""}
    };
    for (int i = 0; i < 7; i++) {
        printf("%s ", arr[i][0]);
        scanf(" %[^\n]s", arr[i][1]);
    }
    system("clear");
    for (int i = 0; i < 7; i++) {
        printf("%s %s\n", arr[i][0], arr[i][1]);
    }
    return 0;
}

File: Modul 02/01/PRAK201-2410817310020-MRizkiDinar.py
import os

def cls():
    os.system('cls' if os.name=='nt' else 'clear')

arr = [
    ["Nama                      :", 0],
    ["NIM                       :", 0],
    ["Kelas Paralel             :", 0],
    ["Tempat/Tanggal Lahir      :", 0],
    ["Alamat                    :", 0],
    ["Hobby                     :", 0],
    ["No. HP                    :", 0]
]
for i in range(len(arr)):
    arr[i][1] = input(arr[i][0])
cls()
for i in range(len(arr)):
    print(arr[i][0] + arr[i][1])